packages/run-protocol/src/vaultFactory/vaultManager.js

```diff
@@ -342,12 +337,27 @@ export const makeVaultManager = (
 
   observeNotifier(periodNotifier, timeObserver);
 
+  const mintAndReallocate = (toMint, fee, seat, ...otherSeats) => {
+    mintAndReallocateWithFee(toMint, fee, seat, ...otherSeats);
+    totalDebt = AmountMath.add(totalDebt, toMint);
+    // TODO signal updater?
```

Comment:
Good question. We do "track" total debt but I suppose there are two levels of tracking: available or notified. I don't know of use cases to be notified of increases in total debt of a vault manager. What are some?

Reply:
This would push the updated total out to the external DB for apps to query. Not urgent

Reply:
should have a ticket link for resolving the TODO (either by doing it or deciding it need not be done)